src/lib/client/Client.ts

```diff
@@ -130,14 +181,31 @@ export class Client extends BaseClient {
 		this.registerStore(this.events)
 			.registerStore(this.actions);
 
-		const coreDirectory = join(__dirname, '../');
+		const coreDirectory = join(__dirname, '../../');
 		for (const store of this.pieceStores.values()) store.registerCoreDirectory(coreDirectory);
 
 		for (const plugin of Client.plugins) plugin.call(this);
 	}
 
+	/**
+	 * Returns a new Cache of all channels.
+	 * @since 0.0.1
+	 */
+	get channels(): Cache<string, Channel> {
+		return new Cache<string, Channel>().concat(this.dms, ...this.guilds.map(guild => guild.channels));
+	}
+
+	/**
+	 * Returns a new Cache of all guild emojis.
+	 * @since 0.0.1
+	 */
+	get emojis(): Cache<string, GuildEmoji> {
+		return new Cache<string, GuildEmoji>().concat(...this.guilds.map(guild => guild.emojis));
```

Comment:
Why is the `<string, GuildEmoji>` needed here?

Reply:
Type 'Cache<unknown, unknown>' is not assignable to type 'Cache<string, GuildEmoji>'.